lib/sse_consumer.ex

```diff
@@ -121,10 +134,13 @@ defmodule SSEConsumer do
 
   def handle_info(%HTTPoison.AsyncChunk{chunk: chunk}, %State{} = state) do
     chunk = state.remaining_stream <> chunk
+
     case ServerSentEvent.parse_all(chunk) do
       {:ok, {events, remaining_stream}} ->
         handle_events(events, state)
+        {:ok, _} = HTTPoison.stream_next(state.async_ref)
```

Comment:
Might be worth it to put `stream_next` before `handle_events`, to do some work in parallel.

Is there a good reason not to do it?